testcafeRunner.js

```diff
@@ -15,12 +21,12 @@ createTestcafe('localhost')
     server = app.listen('1234');
 
     return tc.createRunner()
-      .src(['**/*ui.test.js'])
-      .browsers('chrome:headless')
+      .src([testsToRun])
+      .browsers(browserToRun)
       .concurrency(1)
       .reporter(['spec', {
-          name: 'nunit',
-          output: 'integration-test-report.xml',
+        name: 'nunit',
```

Comment:
can there be something that changes this too so that it only generates the xml file on test?

Reply:
We also have this on our radar. 
https://buyingcatalog.visualstudio.com/Buying%20Catalogue/_workitems/edit/3759